tests/testthat/test-vendor-params.R

```diff
@@ -0,0 +1,11 @@
+test_that("vendor param count works", {
```

Comment:
why are there new fixtures, and what is their size?

Reply:
Because I'm testing a new request. Small because it only returns the first feature

Reply:
but there is no httptest::with_mock_dir in this test?

Reply:
Oh yesh. Actually it doesn't seem like there is fixture for it but there probably should be